repos/system_upgrade/el7toel8/actors/kernel/checkkernelmodules/actor.py

```diff
@@ -0,0 +1,52 @@
+from leapp.actors import Actor
+from leapp.libraries.common.reporting import report_with_remediation
+from leapp.models import ActiveKernelModulesFacts, WhitelistedKernelModules
```

Comment:
It would be nice to rename the ActiveKernelModulesFacts to LoadedKernelModulesFacts.

Reply:
We can do this in the separate PR. This will require changes across multiple actors.